srcpkgs/zzz-user-hooks/template

```diff
@@ -0,0 +1,25 @@
+# Template file for 'zzz-user-hooks'
+pkgname=zzz-user-hooks
+version=1.0.0
+revision=1
+build_style=gnu-makefile
+checkdepends="shellcheck"
```

Comment:
`depends="bash sudo"`

---
Below are not comment on template, but the code itself

- `bash`: It looks like they're using non-required bashism. With a quick skim over the code, the code could be rewritten in POSIX sh.
- `sudo`: I prefer `runuser` or `chpst` instead.

Reply:
I agree with the bashisms but it seems the maintainer likes bashisms according to his blog.

I have opened an issue asking to change to runuser or chpst (https://github.com/bahamas10/zzz-user-hooks/issues/5).